src/board/v3/platform_signal.c

```diff
@@ -25,3 +27,11 @@ void platform_decode_setup(void)
 {
   decode_gps_l1ca_register();
 }
+
+void platform_ndb_init(void)
+{
+  ndb_init();
+  ndb_p1_init();
```

Comment:
Just call the individual ndb_\<type\>_init() functions directly here. Keep the board-specific abstraction in this file.